install_config/install/advanced_install.adoc

```diff
@@ -20,15 +20,18 @@ own implementation using the configuration management tool of your choosing.
 Alternatively, you can use the link:quick_install.html[quick installation]
 method for trial installations.
 
-== Prerequisites
+== Before You Begin
 
-Before installing OpenShift, you must first link:prerequisites.html[satisfy the
-prerequisites] on your hosts, which includes verifying system and environment
-requirements per component type and properly installing and configuring Docker.
+Before installing OpenShift, you must first see the
+link:prerequisites.html[Prerequisites] topic to prepare your hosts, which
+includes verifying system and environment requirements per component type and
+properly installing and configuring Docker. It also includes installing Ansible
+version 1.8.4 or later, as the advanced installation method is based on Ansible
+playbooks and as such requires directly invoking Ansible.
 
 After following the
 instructions in the link:prerequisites.html[Prerequisites] topic, you can
-continue to link:#installing-ansible[installing Ansible].
+continue to link:#installing-ansible[Installing Ansible].
 
 [[installing-ansible]]
```

Comment:
You can drop the "Installing Ansible" section now because it is now installed automatically when you install atomic-openshift-utils in the Prerequisites topic. After removing this section, you can touch up the end off the last section to end with "you can continue in this topic to Configuring Ansible" instead of linking to "Installing Ansible". Also check the repo for other hits on `#installing-ansible` to updated those (I only found one other one, at the end of the Prerequisites topic, which can now link directly to `#configure-ansible` instead).